src/main/drivers/pwm_stm32f4xx.c

```diff
@@ -31,6 +31,10 @@
 
 #define MAX_DMA_TIMERS 8
 
+#define MOTOR_BIT_0 17
```

Comment:
Still needs to be verified. These do work, but I am curious to the explaination of why it didnt't match.

@ledvinap Do you have any idea why the old values 15 and 30 were giving 680khz instead of desired 600? The current values seem to do 580, which I tested fine.